guides/controllers.md

```diff
@@ -22,7 +22,7 @@ end
 
 The first line below the module definition invokes the `__using__/1` macro of the `HelloWeb` module, which imports some useful modules.
 
-The `PageController` gives us the `index` action to display the Phoenix [welcome page] associated with the default route Phoenix defines in the router.
+The `PageController` controller gives us the `index` action to display the Phoenix [welcome page] associated with the default route Phoenix defines in the router.
```

Comment:
Alternatively it can be expressed like this:
```suggestion
`PageController` gives us the `index` action to display the Phoenix [welcome page] associated with the default route Phoenix defines in the router.
```